Marlin/src/module/planner.h

```diff
@@ -512,6 +527,11 @@ class Planner {
     #if ENABLED(LIN_ADVANCE)
       static float extruder_advance_K[DISTINCT_E];
     #endif
+    #if ENABLED(SMOOTH_LIN_ADV)
+      static float extruder_advance_TAU,
+                   extruder_advance_TAU_TICKS,
+                   extruder_advance_ALPHA;
```

Comment:
These variables are not used in the planner code. I think they should be moved to the stepper module.

Reply:
I had just put them next to `extruder_advance_K` without thinking about it.

Moved now